cobigen-templates/templates-devon4j/src/main/templates/crud_openapi_angular_service_based_app/templates.xml

```diff
@@ -6,7 +6,7 @@
     <templateExtension ref="${variables.component#lower_case}RestController.service.ts" mergeStrategy="tsmerge"/>
     <templateExtension ref="api.ts" mergeStrategy="tsmerge"/>
     <templateExtension ref="environment.ts" mergeStrategy="tsmerge_override"/>
-    <templateExtension ref="environment.prod.ts" mergeStrategy="tsmerge_override"/> 
+    <templateExtension ref="environment.prod.ts" mergeStrategy="tsmerge_override"/>
   </templates>
 
   <templateScans>
```

Comment:
I think you forgot to pull the latest changes from CobiGen master. This file is outdated, it should not be modified.